EIPS/eip-ethereum_message_signing.md

```diff
@@ -0,0 +1,57 @@
+## Preamble
+
+    EIP: <to be assigned>
+    Title: Ethereum message signing
+    Author: ligi <ligi@ligi.de>
+    Type: Standard Track
+    Category: ERC
+    Status: Draft
+    Created: 2017-08-09
+
+## Simple Summary
+
+Standard for message signing with Ethereum accounts.
+
+## Abstract
+
+Messages should not be signed in a raw form directly to prevent unintended signing e.g. of valid transactions. We use a prefix here to prevent the data we sign to be something like a valid transaction.
+
+## Motivation
+
+The need for this EIP came up in a [github issue of go-ethereum](https://github.com/ethereum/go-ethereum/issues/14794). As there was no EIP for this use-case, different incompatible implementations for this use-case emerged. This EIP is intended to formalize how messages are prefixed so implementations for this use-case can be compatible
+
+## Specification
+
+`<varint_prefix_length><prefix><varint_message_length><message>`
```

Comment:
The prefix should be included in this EIP.

Reply:
I disagree strongly here